aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -108,7 +108,7 @@ module aptos_framework::stake {
     /// Having this be separate from the signer for the account that the validator resources are hosted at allows
     /// modules to have control over a validator.
     struct OwnerCapability has key, store {
-        pool_address: address,
+        pool_address: address
```

Comment:
Curious: why so many format changes? 🤔

Reply:
it's ide behavior... so I split it into different commits.